CONTRIBUTING.md

```diff
@@ -93,8 +93,12 @@ The FreeCAD Contribution Process is expressed here with the following specific g
 
 ## 8. Project Administration
 
-1. The project Administrators will manage the set of project Maintainers.  They SHALL maintain a sufficiently large pool of Maintainers to ensure their succession and permit timely review of contributions.
-2. Contributors who have a history of successful PRs and have demonstrated continued professionalism should be invited to be Maintainers.
-3. Administrators SHOULD remove Maintainers who are inactive for an extended period, or who repeatedly fail to apply this process accurately.
-4. The list of Maintainers SHALL be publicly accessible and reflective of current activity on the project.
-5. Administrators SHOULD block or ban “bad actors” who cause stress, animosity, or confusion to others in the project. This SHOULD be done after public discussion, with a chance for all parties to speak. A bad actor is someone who repeatedly ignores the rules and culture of the project, who is hostile or offensive, who impedes the productive exchange of information, and who is unable to self-correct their behavior when asked to do so by others.
+1. Project Administrators are those individuals who are members of the FreeCAD Github organization and have the role of 'owner'.  They have the task of administering the organization including adding and removing individuals from various teams.
```

Comment:
More readable perhaps? Also avoid the word 'owner':
Project Administrators are members of the FreeCAD GitHub organization whose task it is to administer the organization, including adding and removing individuals from various teams.

Reply:
The 'owner' role comes from GitHub. These people own the repo. Not our language.

Reply:
I like @Roy-043 's description. Should we use github's terms? What if we change to another platform? What about identifying administrators more clearly on all platforms? (Ex. on the forum too).